src/applications/fry-dea/components/ConfirmationResponses.jsx

```diff
@@ -152,6 +152,108 @@ UnderReview.prototype = {
   response: PropTypes.object || PropTypes.bool,
 };
 
+export function Approved(name) {
+  return (
+    <>
+      <div>
+        <va-alert
+          close-btn-aria-label="Close notification"
+          status="success"
+          visible
+        >
+          <h2 id="track-your-status-on-mobile" slot="headline">
```

Comment:
Do we need this ID or was it just copied and pasted? If we don't need it, please remove it.

Reply:
resolved